chanel/urls.py

```diff
@@ -1,19 +1,5 @@
-"""chanel URL Configuration
-
-The `urlpatterns` list routes URLs to views. For more information please see:
-    https://docs.djangoproject.com/en/3.0/topics/http/urls/
-Examples:
-Function views
-    1. Add an import:  from my_app import views
-    2. Add a URL to urlpatterns:  path('', views.home, name='home')
-Class-based views
-    1. Add an import:  from other_app.views import Home
-    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
-Including another URLconf
-    1. Import the include() function: from django.urls import include, path
-    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
-"""
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
+    path('product/', include('products.urls')),
```

Comment:
settings 내부에 있는 url에는 앞뒤로 모두 슬래시를 빼는 것이 wecode coding convention입니다 :)
```suggestion
    path('product', include('products.urls')),
```